packages/crypto/src/pbkdf2.ts

```diff
@@ -11,9 +11,9 @@ import { sha512 as nobleSha512 } from "@noble/hashes/sha512";
  */
 export async function getCryptoModule(): Promise<any | undefined> {
   try {
-    const crypto = await import("crypto");
+    const crypto = await require("crypto");
```

Comment:
Could you try this instead?

```
    // HACK: Use a variable to get webpack to ignore this and cause a
    // runtime error instead of build system error or fallback implementation.
    const nodeCryptoPackageName = "crypto";
    const crypto = await import(nodeCryptoPackageName);
 ```
 
 We use this in a different place already. This should avoid build systems trying to bundle a crypto implementation when it is not available natively.

Reply:
I introduced this, however the problem is the same. It attempts to resolve `crypto` when bundling. In my current project, using Expo/React-Native/Metro, it seems to parse resolve any `import` call, hence why I switched it to a `require`. I believe this is specific to the Metro bundler. 

Any ideas on how to solve this?